dev/pyproject.py

```diff
@@ -89,6 +89,12 @@ def build(skinny: bool) -> None:
                     # Required for exporting metrics from the MLflow server to Prometheus
                     # as part of the MLflow server monitoring add-on
                     "prometheus-flask-exporter",
+                    # Required to use MySQL as the backend store
```

Comment:
It would be awesome to get this merged. See:
* https://github.com/bitnami/charts/pull/25965
* https://github.com/bitnami/charts/issues/22992
* https://github.com/bitnami/charts/issues/25964

Maybe @harupy or @dbczumar or @BenWilson2 could triage? Thanks!